.eslintrc.json

```diff
@@ -19,7 +19,8 @@
     "react/forbid-prop-types": 0,
     "react/jsx-props-no-spreading": 0,
     "import/no-extraneous-dependencies": [2, { "devDependencies": true }],
-    "prefer-destructuring": ["error", { "object": true, "array": false }]
+    "prefer-destructuring": ["error", { "object": true, "array": false }],
+    "no-param-reassign": [2, { "props": false }]
```

Comment:
what does 2 mean here?

Reply:
found it https://eslint.org/docs/2.0.0/rules/no-param-reassign, but it does not explain that...

Reply:
Don't know honestly, I copy pasted it and made sure it works. Still gives error if I would reassign the entire arg (`someArg = newValue`).

Reply:
> Don't know honestly, I copy pasted it and made sure it works. Still gives error if I would reassign the entire arg (`someArg = newValue`).

Never mind. Cool.